src/components/Modal/VotingPowerDelegationModal/VotingPowerDelegationCandidatesList.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+
+import Markdown from 'decentraland-gatsby/dist/components/Text/Markdown'
+import useAuthContext from 'decentraland-gatsby/dist/context/Auth/useAuthContext'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { Button } from 'decentraland-ui/dist/components/Button/Button'
+import { Close } from 'decentraland-ui/dist/components/Close/Close'
+import { Modal, ModalProps } from 'decentraland-ui/dist/components/Modal/Modal'
+
+import { CANDIDATE_ADDRESSES } from '../../../constants'
+import { EDIT_DELEGATE_SNAPSHOT_URL } from '../../../entities/Proposal/utils'
+import useDelegatesInfo, { Delegate } from '../../../hooks/useDelegatesInfo'
+import useVotingPowerDistribution from '../../../hooks/useVotingPowerDistribution'
+import DelegatesTable from '../../Table/DelegatesTable'
+
+import './VotingPowerDelegationCandidatesList.css'
+
+type VotingPowerDelegationCandidatesListProps = Omit<ModalProps, 'children'> & {
+  setSelectedCandidate: (candidate: Candidate) => void
+  showPickOtherDelegateButton?: boolean
+}
+
+export type Candidate = Delegate & {
+  bio: string
+  links: string[]
+  relevant_skills: string[]
+  involvement: string
+  motivation: string
+  vision: string
+  most_important_issue: string
+}
+
+function VotingPowerDelegationCandidatesList({
+  onClose,
+  setSelectedCandidate,
+  showPickOtherDelegateButton,
+  ...props
+}: VotingPowerDelegationCandidatesListProps) {
+  const delegates = useDelegatesInfo(CANDIDATE_ADDRESSES)
+  const [userAddress] = useAuthContext()
+  const { vpDistribution } = useVotingPowerDistribution(userAddress)
+
+  const t = useFormatMessage()
+
+  return (
+    <Modal
+      {...props}
+      onClose={onClose}
+      size="small"
+      closeIcon={<Close />}
+      className="GovernanceContentModal VotingPowerDelegationCandidatesList"
+    >
+      <Modal.Header className="VotingPowerDelegationCandidatesList__Header">
+        {t('modal.vp_delegation.title')}
+      </Modal.Header>
+      <Modal.Description className="VotingPowerDelegationCandidatesList__Description">
+        <Markdown>
+          {(userAddress && vpDistribution
+            ? t('modal.vp_delegation.description', { vp: vpDistribution.own })
+            : t('modal.vp_delegation.description_generic')) +
```

Comment:
Are we using these two keys in other places? Or maybe we can use only one key instead of two with a space?